config/rules.toml

```diff
@@ -1,17 +1,49 @@
 [[rule]]
+    # Only one regex has to match to send the record. Order does not matter.
     regexs = [
-        '.*a.*'
+        '.*a.*',
+        '^abc\.def\.'
     ]
     clusters = [
+        # Matching records will be sent here.
         'local0'
     ]
+    # If true, even if there is a match, we continue to the next rule.
     continue = true
 
 [[rule]]
+    # Prefix matching can be used instead of regex to match prefixes.
+    # Prefix matching is faster than prefix matching.
```

Comment:
typo. 
Prefix matching is faster than regex matching.

Reply:
Done.